src/lib/__tests__/__cardFunctionsTests__/cardNumber.test.js

```diff
@@ -0,0 +1,12 @@
+const { cardNumber } = require("lib/cardFunctions");
+
+describe("cardNumber function", () => {
+  test("should return 08", () => {
+    //GIVEN
+    const card = "C08";
+    //WHEN
+    const result = cardNumber(card);
+    //THEN
+    expect(result).toEqual("08");
```

Comment:
```suggestion
    expect(result).toBe("08");
```
toEqual should be better be used for complex data structures content like objects and arrays